webpack.config.js

```diff
@@ -56,16 +23,22 @@ var config = {
         exclude: /node_modules/,
         loader: "babel-loader",
         query: {
-          presets: ["react", "es2015"]
+          presets: ["react"]
         }
       }
     ],
   },
 
   plugins: [
     new CopyWebpackPlugin([
-      {from: "**/*"},
-    ], {ignore: ignoreOnCopy}),
+      {from: "bootstrap.js"},
+      {from: "chrome.manifest"},
+      {from: "install.rdf"},
+      {from: "webextension/manifest.json", to: "webextension/"},
+      {from: "webextension/en-US.ftl", to: "webextension/"},
+      {from: "webextension/manage.html", to: "webextension/"},
```

Comment:
would it make sense to change this to "webextension/*.html"?

I'm worried about inadvertently forgetting files.